test/uk/gov/hmrc/merchandiseinbaggage/CoreTestData.scala

```diff
@@ -304,14 +311,14 @@ trait CoreTestData {
         declarationType,
         maybeGoodsDestination.get,
         goodsEntries.declarationGoodsIfComplete.get,
-        maybeNameOfPersonCarryingTheGoods.get,
+        maybeNameOfPersonCarryingTheGoods.getOrElse(Name("xx", "yy")),
```

Comment:
finally, we have decided that `.get` is a very bad idea :-)

Reply:
it's test code....so kind of allowed.....but yes better without